dataservice/api/common/schemas.py

```diff
@@ -111,10 +111,20 @@ def check_unknown_fields(self, data, original_data):
             raise ValidationError('Unknown field', unknown)
 
 
+def acl_deprecation(v):
+    if v is not None and not v == []:
```

Comment:
Couldn't this just be `if v`?

Reply:
What if `v = False`?

Reply:
sure, or "", but can v be those things?

Reply:
v is only allowed to be `[]` or undefined (`None`)

Reply:
before or after this point? if before then if v would be ok

Reply:
`v` comes from the request body that the user sends us, so it _can_ be anything.
This function defines the constraint that we want to enforce (`v ∈ ([], None)`)

Reply:
Is `if v is not None and not v == []` = `not (v is None or v is [])` right?....De morgans or whatever that boolean negation law is lol? If yes, then I think this PR is ok